src/test/resources/org/elm/lang/core/parser/fixtures/partial/Strings.txt

```diff
@@ -0,0 +1,59 @@
+Elm File
+  ElmValueDeclaration(VALUE_DECLARATION)
+    ElmFunctionDeclarationLeft(FUNCTION_DECLARATION_LEFT)
+      PsiElement(LOWER_CASE_IDENTIFIER)('a')
+    PsiElement(EQ)('=')
+    ElmExpression(EXPRESSION)
+      ElmCharConstant(CHAR_CONSTANT)
+        PsiElement(CHAR_LITERAL)(''c')
+  PsiElement(VIRTUAL_END_DECL)('\n')
+  PsiWhiteSpace('\n')
+  ElmValueDeclaration(VALUE_DECLARATION)
+    ElmFunctionDeclarationLeft(FUNCTION_DECLARATION_LEFT)
+      PsiElement(LOWER_CASE_IDENTIFIER)('b')
+    PsiElement(EQ)('=')
+    ElmExpression(EXPRESSION)
+      ElmCharConstant(CHAR_CONSTANT)
+        PsiElement(CHAR_LITERAL)(''d')
+  PsiElement(VIRTUAL_END_DECL)('\n')
+  PsiWhiteSpace('\n')
+  ElmValueDeclaration(VALUE_DECLARATION)
+    ElmFunctionDeclarationLeft(FUNCTION_DECLARATION_LEFT)
+      PsiElement(LOWER_CASE_IDENTIFIER)('c')
+    PsiElement(EQ)('=')
+  PsiElement(OPEN_QUOTE)('"')
```

Comment:
I was hoping to be able to still parse these as a string constants, rather than just unexpected tokens, but no amount of fiddling with `pin` and `recoverWhile` got that to work. Do you of know a way to do it?